dialogue-serde/src/main/java/com/palantir/conjure/java/dialogue/serde/ConjureBodySerDe.java

```diff
@@ -172,7 +172,7 @@ public RequestBody serialize(T value) {
             return new RequestBody() {
 
                 @Override
-                public void writeTo(OutputStream output) {
+                public void writeTo(OutputStream output) throws IOException {
                     encoding.serializer.serialize(value, output);
```

Comment:
For documenting: the actual retries here will now properly happen in RetryingChannel#shouldAttemptToRetry, because we only retry IOExceptions. Previously the underlying IOException was wrapped, and thus not retried.